python/mxnet/ndarray_doc.py

```diff
@@ -54,6 +55,274 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class ConcatDoc(NDArrayDoc):
+    """
+    Examples
+    --------
+
+    Joins input arrays along a given axis.
+
+    >>> x = mx.nd.array([[1,1],[2,2]])
+    >>> y = mx.nd.array([[3,3],[4,4],[5,5]])
+    >>> z = mx.nd.array([[6,6], [7,7],[8,8]])
+
+    >>> mx.nd.concat(x,y,z,dim=0)
```

Comment:
I'm not seeing this show up in the preview...

Reply:
I do see it. Did you look at the right one? 

http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-14243/7/api/python/ndarray/ndarray.html#mxnet.ndarray.Concat

The preview link is versioned, as mentioned in the pull request checklist: 
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html

Reply:
Why does the previous API render the interpreter prompt, but this one does not? It looks close, but I think they're different because it is missing these:
   **>>>** y = mx.nd.array([[3,3],[4,4],[5,5]])

Reply:
Any idea why?

Reply:
<img width="806" alt="Screen Shot 2019-05-11 at 11 35 36 AM" src="https://user-images.githubusercontent.com/5545640/57573712-fa41de00-73e0-11e9-8530-b4831524746e.png">

> It looks close, but I think they're different because it is missing these:
> \>>> y = mx.nd.array([[3,3],[4,4],[5,5]])

I did find this. What do you think is missing????

Reply:
Ya, I guess he was looking at some other preview doc.